### ISSUE TITLE
Travis CI fail wordpress-coding-standards

### ISSUE THREAD

**Issue by roby94** [olefredrik/FoundationPress#985]
Hi,

It's my first time using using Travis CI, so sorry for the stupid question.
I've created a fork of FoundationPress, created a new brach and commited sono new code, but all test in travis failed with the following error:

```
$ scripts/phpcs --config-set installed_paths ../wordpress-coding-standards
/home/travis/.travis/job_stages: line 54: scripts/phpcs: No such file or directory

The command "scripts/phpcs --config-set installed_paths ../wordpress-coding-standards" failed and exited with 127 during .

Your build has been stopped.
```

How can i fix this?
Thank you

**Comment by linuxbastard** [comment]
Try this:
[PHPCS not running on the command line?](http://stackoverflow.com/a/11787140/1467989)

**Comment by roby94** [comment]
Thank you @linuxbastard 
resolved with #987
